epub33/core/index.html

```diff
@@ -181,23 +181,27 @@ <h3>Relationship to Other Specifications</h3>
 
 				<section id="sec-overview-relations-html">
 					<h4>Relationship to HTML</h4>
-					<p>This specification does not reference a specific version of W3C [[HTML]], but instead uses an
-						undated reference that will always point to the latest recommendation. This approach ensures
-						that EPUB will always keep pace with changes to the HTML standard. Authors will need to keep
-						track of changes to HTML, and ensure that their processes are kept up to date.</p>
 
-					<div class="caution">
+					<p>The [[HTML]] standard is continuously evolving &#8212; there are no longer versioned releases of
+						it. That standard, in turn, references various technologies that continue to evolve, such as
+						MathML, SVG, CSS and JavaScript.</p>
+
```

Comment:
"SVG, CSS and Javascript" -> "SVG, CSS, and Javascript"

(In general, the W3C house style favors Oxford commas. This may require a more thorough editorial review...)

Reply:
Heh, ISO rules picked the other way at these and "Web" in the a11y doc. Round and round...

We never bothered with serial commas in IDPF. Unless there's some particular ambiguity the lack of one introduces, there's no harm. It will take some review if we need to make usage consistent, as I certainly wasn't adding and I wasn't fixing in any other contributions.

Web is probably capitalized, though, as that was more a slip-up from still thinking in ISO terms.

Reply:
> Heh, ISO rules picked the other way at these and "Web" in the a11y doc. Round and round...

One standard is good, more is better :-)